app/services/geocode_service.rb

```diff
@@ -2,7 +2,7 @@ class GeocodeService
   attr_reader :connection
 
   def initialize
-    @connection ||= Faraday.new(url: "https://maps.googleapis.com/maps/api/geocode")
+    @connection = Faraday.new(url: "https://maps.googleapis.com/maps/api/geocode")
```

Comment:
the geocode_info method allows us to make different calls.  I don't think the url in initialize changes.